tests/base.py

```diff
@@ -239,7 +240,7 @@ def tearDown(self):
     httpretty.DELETE,
 ]
 def kill(*args, **kwargs):
-    print('httppretty.kill', args, kwargs)
+    logger.error('httppretty.kill: %s - %s', args, kwargs)
```

Comment:
passing as args should str them, just verified in ipython, plus it lazy evals based on the set log level, where `.format` can format and then potentially never display the string.